.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+#### What does this PR do?
+
+#### Description of Task to be completed?
+
+#### How should this be manually tested?
+
+#### Any background context you want to provide?
+
+#### What are the relevant pivotal tracker stories?
+
+#### Screenshots (if appropriate)
+
+#### Questions:
```

Comment:
Thanks for setting the PR template! Great job. 
I would implore us though to explore any merits of setting commit message templates locally on individual machines as these could then translate directly into PRs